rocks/designate-consolidated/README.md

```diff
@@ -17,16 +17,16 @@ it will help ensure that all layers of the image are imported
 into docker (this is just the top layer).
 
 ```bash
-> skopeo --insecure-policy copy oci-archive:designate-consolidated_2024.1_amd64.rock docker-daemon:designate-consolidated:2024.1
+> skopeo --insecure-policy copy oci-archive:designate-consolidated_2025.1_amd64.rock docker-daemon:designate-consolidated:2025.1
 ```
 
 If you are interested in giving it a go in Microk8s, you can
 export the image from your docker registry and then into the
 microk8s registry:
 
 ```bash
-> docker save designate-consolidated:2024.1 > ./designate-consolidated_2024.1.tar
-> microk8s ctr image import ./designate-consolidated_2024.1.tar
+> docker save designate-consolidated:2025.1 > ./designate-consolidated_2025.1.tar
+> microk8s ctr image import ./designate-consolidated_2025.1.tar
 # Try with sunbeam
-> juju attach-resource designate-consolidated designate-consolidated-image=designate-consolidated:2024.1
+> juju attach-resource designate-consolidated designate-consolidated-image=designate-consolidated:2025.1
```

Comment:
same comment as barbican

Reply:
I think this should be part of another change that we can backport to stable/2024.1